cilium/cmd/monitor.go

```diff
@@ -59,40 +63,97 @@ var (
 		SampleType:   bpf.PERF_SAMPLE_RAW,
 		WakeupEvents: 1,
 	}
+	eventTypeIdx = -1 // for integer comparison
+	eventType    = ""
+	eventTypes   = []string{"drop", "debug", "capture"}
```

Comment:
This also makes it clear which names corresponds to which message type without depending on order